### ISSUE TITLE
Description

### ISSUE THREAD

**Issue by rtalexander** [jugyo/SublimeSimpleMarker#1]
Please consider adding a description of what SimpleMarker does, how its used, etc. The video is nice, it's not clear (at least to me) what the intended use is or its benefits.

**Comment by eorroe** [comment]
Agreed. Manipulation of the markers would be nice. It seems like it's just hightlighting the markers. Which [HightlightWords Package](https://packagecontrol.io/packages/HighlightWords) does better.

Now It would be useful If I can manipulate the markers. Like multi select, delete, ....... and more.
